### PR TITLE
added wise quotes

### PR DESCRIPTION
**Pull request by jyotijangid**
#33

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -117,7 +118,9 @@
     "Anything is possible when you have inner peace - Master Shifu",
     "All we have to decide is what to do with the time that is given to us - Gandalf the Grey",
     "Respect your efforts, respect yourself. Self-respect leads to self-discipline. When you have both firmly under your belt, that’s real power – Clint Eastwood",
-    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent"
+    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent",
+    "It's the possibility of having a dream come true that makes life interesting. - Paulo Ceolho, The Alchemist",
+    "The only way to learn is to live. - Matt Haig, The Midnight Library
```
Comment (akashsara) [inline config.py:123]: Formatting needs a fix.

File: config.py [hunk 2]
```diff
@@ -86,6 +86,7 @@
     "Don't count on it.",
     "My sources say no.",
     "Don't try to play the fool with me Nikesh.",
+    "I wouldn't mind"
```
Comment (akashsara) [inline config.py:89]: Please remove this.

**Review by akashsara — CHANGES_REQUESTED** [review]

**Comment by akashsara** [comment]
Hi, please refer to the review. Also, please add a few more quotes.

File: config.py [hunk 3]
```diff
@@ -117,7 +118,15 @@
...
     "Respect your efforts, respect yourself. Self-respect leads to self-discipline. When you have both firmly under your belt, that’s real power – Clint Eastwood",
-    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent"
+    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent",
+    "It takes a great deal of bravery to stand up to our enemies, but just as much to stand up to our friends. - Prof Albus Dumbledore",
+    "It's the possibility of having a dream come true that makes life interesting. - Paulo Ceolho, The Alchemist",
+    "The only way to learn is to live. - Matt Haig, The Midnight Library",
+    "If you can't explain it to a six year old, you don't understand it yourself. - Albert Einstein",
+    "Happiness can be found, even in the darkest of times, if one only remembers to turn on the light. - Prof Albus Dumbledore",
+    "You must be the change you wish to see in the world. - Mahatma Gandhi",
+    "“Let us always meet each other with a smile, for the smile is the beginning of love. - Mother Teresa”,
```
Comment (akashsara) [inline config.py:128]: Syntax error

**Review by akashsara — CHANGES_REQUESTED** [review]

File: config.py (hunk 3 above)
Comment (jyotijangid) [reply config.py]: @akashsara removed the syntax error. thankyou

**Comment by akashsara** [comment]
Please pull the latest changes and resubmit your PR.

**Comment by jyotijangid** [comment]
branch updated @akashsara

File: config.py [hunk 4]
```diff
@@ -117,7 +118,16 @@
...
+    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent",
+    "It takes a great deal of bravery to stand up to our enemies, but just as much to stand up to our friends. - Prof Albus Dumbledore",
+    "It's the possibility of having a dream come true that makes life interesting. - Paulo Ceolho, The Alchemist",
+    "The only way to learn is to live. - Matt Haig, The Midnight Library",
+    "If you can't explain it to a six year old, you don't understand it yourself. - Albert Einstein",
+    "Happiness can be found, even in the darkest of times, if one only remembers to turn on the light. - Prof Albus Dumbledore",
+    "You must be the change you wish to see in the world. - Mahatma Gandhi",
+    "Let us always meet each other with a smile, for the smile is the beginning of love. - Mother Teresa",
+    "A dream is not that which you see while sleeping, it is something that does not let you sleep. - Dr APJ Abdul Kalam",
+    "You either die a hero, or live long enough to see yourself become the villain - Harvey Dent",
```
Comment (akashsara) [inline config.py:130]: Duplicate (see line 121)

**Review by akashsara — CHANGES_REQUESTED** [review]

**Comment by akashsara** [comment]
Thanks for all the work @jyotijangid! There's just one small issue in that you've added the same quote twice. Once you fix that I'll merge it in.

**Comment by jyotijangid** [comment]
@akashsara  removed the quote. 
Thankyou for your patience.

**Pull request merged**